mobile/src/main.jsx

```diff
@@ -50,6 +52,10 @@ const renderAppWithPersistedState = persistedState => {
     }
   }
 
+  document.addEventListener('resume', () => {
```

Comment:
It comes from cordova ⇒ https://cordova.apache.org/docs/en/latest/cordova/events/events.html#resume